src/browserbehavior/ExtendedBrowserBehavior.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { BrowserBehavior } from '..';
+
+/**
+ * This interface contains methods on {@link DefaultBrowserBehavior} that were
+ * incorrectly omitted from {@link BrowserBehavior}, plus new methods that
+ * were added since the last major version.
+ *
+ * Members of this interface can change without a major version bump to accommodate new browser
```

Comment:
Could you elaborate on what this means? Aren't we allowed to modify any interface before a major version bump?

Reply:
This is a note to customers warning them that this interface can change without a major version bump.